doc/api/child_process.markdown

```diff
@@ -755,7 +784,7 @@ grep.stdin.end();
 * `message` {Object}
 * `sendHandle` {Handle object}
 * `callback` {Function}
-* Return: Boolean
+* Return: Boolean | Promise
```

Comment:
Just to clarify, one example of how this could become tricky is that the return value on `child.send()` will be false if the `when the backlog of unsent messages exceeds a threshold that makes it unwise to send more`, which means a developer can currently depend on the return value being falsy to help with throttling or to provide backpressure. If code is sending a bunch of messages and doesn't care about the callback, then a falsy check would fail... 

```
const child = getChildSomehow();
function sendPing() {
  if (child.connected) {
    if (child.send('ping')) {
      setTimeout(sendPing, 1000);
    } else {
      // wait a bit longer before sending again
      setTimeout(sendPing, 2000);   
    }
  }
}
```

The fact that `callback` is already optional on `send` just makes this unpredictable.